mlir/include/mlir/Transforms/GreedyPatternRewriteDriver.h

```diff
@@ -50,24 +50,41 @@ class GreedyRewriteConfig {
   ///
   /// Note: Only applicable when simplifying entire regions.
   bool useTopDownTraversal = false;
+  GreedyRewriteConfig &setUseTopDownTraversal(bool use = true) {
+    useTopDownTraversal = use;
+    return *this;
+  }
```

Comment:
I'm all for encapsulation, but if we're adding methods, can we make all the data members private and turn this into a class?

Reply:
You marked it resolved but I don't see he change in this class?

Reply:
Yeah, pushed the wrong commit...